tests/ClientConfigurationTest.php

```diff
@@ -15,9 +15,9 @@ class ClientConfigurationTest extends TestCase
     public function testInvalidDsn(): void
     {
         $this->expectException(\Elastica\Exception\InvalidException::class);
-        $this->expectExceptionMessage("DSN 'test:0' is invalid.");
+        $this->expectExceptionMessage("DSN ':0' is invalid.");
 
-        ClientConfiguration::fromDsn('test:0');
```

Comment:
Since PHP 8.0 RC1 a 0 port number is considered valid: See https://github.com/php/php-src/pull/6152
For details see how it behaves on other PHP versions: https://3v4l.org/ZYhJc